webapp-django/crashstats/crashstats/form_fields.py

```diff
@@ -1,47 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file contained fields that were used in the past but aren't used anymore--this is all dead code now.